src/rpc/governance.cpp

```diff
@@ -226,6 +230,55 @@ UniValue gobject_prepare(const JSONRPCRequest& request)
 
     return tx->GetHash().ToString();
 }
+
+void gobject_list_prepared_help(CWallet* const pwallet)
+{
+    throw std::runtime_error(
+                "gobject list-prepared <limit>\n"
+                "Returns a list of governance objects prepared by this wallet with \"gobject prepare\" sorted by their creation time.\n"
+                + HelpRequiringPassphrase(pwallet) + "\n"
+                "\nArguments:\n"
+                "1. limit (numeric, optional) Maximal number of objects to return.\n"
+                );
+}
+
+UniValue gobject_list_prepared(const JSONRPCRequest& request)
+{
+    CWallet* const pwallet = GetWalletForJSONRPCRequest(request);
+    if (request.fHelp || (request.params.size() > 2)) {
+        gobject_prepare_help(pwallet);
+    }
+
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp))
+        return NullUniValue;
+
+    EnsureWalletIsUnlocked(pwallet);
+
+    int64_t nLimit = request.params.size() > 1 ? ParseInt64V(request.params[1], "limit") : std::numeric_limits<int64_t>::max();
+    if (nLimit <= 0) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "limit needs to be greater 0");
+    }
+    // Get a list of all prepared governance objects stored in the wallet
+    LOCK(pwallet->cs_wallet);
+    std::vector<const CGovernanceObject*> vecObjects = pwallet->GetGovernanceObjects();
+    // Sort the vector by the object creation time/hex data
```

Comment:
Hmm... I don't think it's that much useful to show `<count>` oldest objects. IMO users would be much more interested in more recent ones and we should sort them the other way around i.e. it should return `<count>` most recent jbjects with the most recent object being the last one in the list, just like `listtransactions` does it for example.

Reply:
Makes sense, see 69d60a4